### DISCUSSION TITLE
Icon Packs & Default Icons do not work properly

### DISCUSSION THREAD

**Discussion by reeceatkinson** [community/community#5195] [Codespaces]
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/78908925/130100454-123919a5-b148-44e1-acd3-9cc0bad01161.png">
The packs work on **reponame**.github.dev however does not work on github.dev/**reponame**

**Comment by reeceatkinson** [comment]
Even with the default the icons still don't show properly:
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/78908925/130102002-a3402a13-d410-4ae4-ac43-022db865608b.png">

**Comment by bamurtaugh** [comment]
cc @misolori for the Icon Packs aspect and @eamodio for "The packs work on **reponame**.github.dev however does not work on github.dev/**reponame**"

**Reply by reeceatkinson** [reply]
Just to clarify the ‘Codespaces icons work’ however the . web editor does not show icon packs properly
@bamurtaugh

**Reply by miguelsolorio** [reply]
Any icon extension (product or file) are currently broken inside of github.dev, subscribe to https://github.com/microsoft/vscode/issues/131248 for an update when that will be fixed

**Reply by reeceatkinson** [reply]
> Any icon extension (product or file) are currently broken inside of github.dev, subscribe to https://github.com/microsoft/vscode/issues/131248 for an update when that will be fixed

It says on the issues that were linked are closed however I am still having issues?

![image](https://user-images.githubusercontent.com/78908925/131560193-1f67d36f-fb21-4cf9-8f56-b8a7e1589c5b.png)

**Reply by miguelsolorio** [reply]
The issue will be updated when it has been fixed, since it's an upstream issue with github.dev it has to be fixed on the GitHub side. Subscribe to be notified when there's an update.
